packages/webcert/src/feature/certificate/Buttons/Test/ReadyForSignButton.test.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { Provider } from 'react-redux'
+import { configureStore, EnhancedStore } from '@reduxjs/toolkit'
+import reducer from '../../../../store/reducers'
+import dispatchHelperMiddleware, { clearDispatchedActions, dispatchedActions } from '../../../../store/test/dispatchHelperMiddleware'
+import { certificateMiddleware } from '../../../../store/certificate/certificateMiddleware'
+import ReadyToSignButton from '../ReadyToSignButton'
+import { readyForSign, updateCertificate } from '../../../../store/certificate/certificateActions'
+import { getCertificate } from '@frontend/common'
+
+const NAME = 'ReadyForSign button name'
+const DESCRIPTION = 'ReadyForSign button description'
+
+describe('ReadyForSign button', () => {
```

Comment:
Stupid comment but file is named Ready**To**SignButton.tsx and test is named Ready**For**SignButton.test.tsx.